### ISSUE TITLE
support for APK splits

### ISSUE THREAD

**Issue by paleozogt** [x2on/gradle-hockeyapp-plugin#71]
In my build I have [ABI splits](http://developer.android.com/intl/zh-cn/google/play/publishing/multiple-apks.html) turned on:

```
splits {
    abi {
        enable true
        universalApk true
    }
}
```

When publishing to hockey, it seems to just pick up the first split it sees (in this case arm64):

```
:MyApp:uploadReleaseToHockeyApp
App file: /Users/dev/workspace/MyAppProj/MyApp/build/outputs/apk/MyApp-master-1.3.5-176-ga4b18a2-b730-arm64-v8a-release.apk
```

I'd like to be able to tell the plugin which split to use (ideally the 'universal' apk).  It doesn't seem like this is doable in v3.3.1.

**Comment by x2on** [comment]
So you like to choose which apk you want to upload, or?

**Comment by paleozogt** [comment]
Ideally, uploading all of the splits would be best.  But short of that, choosing which one (or having it default to the universal apk) would ok.

**Comment by x2on** [comment]
It looks like HockeyApp is not able to support ABI Splits. (http://support.hockeyapp.net/discussions/problems/49508-api-upload-apk-split-version)
But #72 could be a solution.

**Comment by x2on** [comment]
Fixed by #72 in current master
